src/qt/optionsmodel.cpp

```diff
@@ -543,13 +543,19 @@ QVariant OptionsModel::data(const QModelIndex & index, int role) const
             QFont::Weight weight;
             GUIUtil::weightFromArg(settings.value("fontWeightNormal").toInt(), weight);
             int nIndex = GUIUtil::supportedWeightToIndex(weight);
+            if (nIndex == -1) {
+                nIndex = GUIUtil::supportedWeightToIndex(GUIUtil::getSupportedFontWeightNormalDefault());
+            }
             assert(nIndex != -1);
```

Comment:
```diff
-assert(nIndex != -1)
```

It's a bit abundant now

Reply:
Would still prefer to keep it, there is still a conversion going on and if defaults break for some reason, we should know about it.